From 9a71adc7aecbfdf066ba54c763c2ecd8fb09d3cd Mon Sep 17 00:00:00 2001
From: Vikram Bodireddy <vikram.bodireddy@intel.com>
Date: Wed, 6 Feb 2019 15:59:34 +0530
Subject: [PATCH] Selecting 128MB for PFR

PFR platforms requires 128MB flash mapping.
This will override the existing 64MB flash map
and loads 128MB flash map.

Signed-off-by: Vikram Bodireddy <vikram.bodireddy@intel.com>
---
 arch/arm/boot/dts/aspeed-bmc-intel-ast2500.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/aspeed-bmc-intel-ast2500.dts b/arch/arm/boot/dts/aspeed-bmc-intel-ast2500.dts
index 4815104459f1..df02bb1aaf36 100644
--- a/arch/arm/boot/dts/aspeed-bmc-intel-ast2500.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-intel-ast2500.dts
@@ -89,7 +89,7 @@
 	flash@0 {
 		status = "okay";
 		m25p,fast-read;
-#include "openbmc-flash-layout-intel-64MB.dtsi"
+#include "openbmc-flash-layout-intel-128MB.dtsi"
 	};
 };
 
-- 
2.7.4

